lib/themes/dosomething/paraneue_dosomething/js/utilities/ApiClient.js

```diff
@@ -36,6 +43,37 @@ class ApiClient {
   }
 
   /**
+   * Send a DELETE request to the given path URI.
+   *
+   * @param  {String} path
+   * @return {Object}
+   */
+  delete(path) {
+    const url = `${this.url}${path}`;
+
+    // @TODO: see todo in post() method!
+    console.log('%c ApiClient DELETE Request: \n' + url,
```

Comment:
This is what it looks like at the moment on local development:

![image](https://cloud.githubusercontent.com/assets/105849/18008396/ea0e9440-6b75-11e6-9691-115c050c0d34.png)

Will help us while developing to know what is getting passed to API calls via the API client 😄